egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -757,6 +757,10 @@ def __init__(self,
                                  lstm*=0,0.2,0'.  More general should precede
                                  less general patterns, as they are applied
                                  sequentially.""")
+        self.parser.add_argument("--trainer.optimization.alpha", type=float,
```

Comment:
At this level, and probably also on the command line and in the code, best to call this adversarial-training-scale.  'alpha' does not really tell the user what it's all about.